app/controllers/users_controller.rb

```diff
@@ -20,12 +20,12 @@ def create
     end
   end
 
-  def show
-    return unless @user.comments
-    @comments = @user.comments.order_desc.page(params[:page])
-                     .per Settings.comment_items_page
+  def index
+    @users = User.all
```

Comment:
ko dùng all nhé, dùng select kết hợp order

Reply:
Cái của bọn em hơi ít nên em đưa ra là tên người dùng thôi ạ người đăng truyên chứ nó kp tác giả. Phần sau em làm follow thì có thêm lươt follow và show toàn bộ truyện mà ng dừng ý đăng ạ

Reply:
với cả cái này dùng cho admin chứ show ntn hơi bừa ạ. Nên bọn em để phân admin ạ